kitsune/wiki/models.py

```diff
@@ -71,6 +71,16 @@ class Document(NotificationsMixin, ModelBase, BigVocabTaggableMixin, DocumentPer
     # Is this document localizable or not?
     is_localizable = models.BooleanField(default=True, db_index=True)
 
+    restrict_to_staff = models.BooleanField(default=False, db_index=True)
```

Comment:
Copied from the forms.py - Probably this is better suited here

`I would also argue here that if we use a m2m field we can also drop the restrict_to_staff field which is basically a convenience.`